gips/0064-graph-node-version-updates.md

```diff
@@ -0,0 +1,33 @@
+---
+GIP: 0064
+Title: Delegation of responsibility for Graph Node version updates
+Authors: Adam Fuller
+Created: 2024-02-08
+Updated: 2024-02-08
+Stage: Draft
+Category: Process
+---
+
+# Abstract
+
+The Graph Council should delegate responsibility for specification of the protocol-supported version of Graph Node to the Graph Node development team.
+
+# Motivation
+
+This GIP seeks to simplify the release process, and reduce bureaucracy, in order to reduce the time it takes for new features and fixes to be supported on the network by indexers.
+
+# Prior Art
+
+Currently, as specified in GIP-0008, the Graph Council ratifies each Graph Node version via a GGP.
+
+# High-Level Description
+
+The Graph Council should delegate responsibility for specification of the protocol-supported version of Graph Node to the Graph Node core development team. The Graph Council can revoke this delegation of responsibility at any time.
+
+The Graph Node team should provide the Council and community with updates on new versions, along with any salient information regarding functionality or testing.
+
+The Graph Council will still be responsible for enablement of indexing rewards in the feature support matrix, via GGP.
```

Comment:
@azf20, do we want to be this specific, mentioning only "indexing rewards" and not other potentially critical things like arbitration support on specific subgraph features?

Reply:
Makes sense, updated